src/board/system76/common/fan.c

```diff
@@ -105,3 +114,35 @@ uint8_t fan_cooldown(const struct Fan * fan, uint8_t duty) __reentrant {
 
     return highest;
 }
+
+uint8_t fan_smooth(uint8_t last_duty, uint8_t duty) __reentrant {
+  uint8_t next_duty = duty;
```

Comment:
Please indent using 4 columns.

Reply:
@crawfxrd sorry about that, I must have coded most of this prior to installing an editorconfig package for my text editor :/ I'll fix this and others up